script/clone_all_rspec_repos

```diff
@@ -19,6 +19,18 @@ if is_mri; then
   fi
 
   popd
+
+  # We don't want to bundle until all repos have been cloned, so these
+  # come after the clones above.
+  install_bundle_for "rspec"
+  install_bundle_for "rspec-core"
+  install_bundle_for "rspec-expectations"
+  install_bundle_for "rspec-mocks"
+  install_bundle_for "rspec-rails"
+
+  if rspec_support_compatible; then
+    install_bundle_for "rspec-support"
+  fi
```

Comment:
@myronmarston I'm not sure why this part of the diff is different from rspec/rspec-dev#102